### DISCUSSION TITLE
How Ican Interfaces  on status page

### DISCUSSION THREAD

**Discussion by manepape** [immortalwrt/immortalwrt#696] [Q&A]
Hi all,
How to get interfaces on status page of luci theme.
Thanks

![image](https://user-images.githubusercontent.com/106386242/171061790-3072b1db-e9f5-4768-aaf1-dda32955af73.png)

**Comment by 1715173329** [comment]
Install autocore-x86

**Reply by manepape** [reply]
Hi . I installed autocore-x86 but no change.
Does some .lua file need to created?
![image](https://user-images.githubusercontent.com/106386242/171263084-f5650511-5f6e-4884-bbfe-e2a4f55fb952.png)

![image](https://user-images.githubusercontent.com/106386242/171262597-b78ce1bc-5a8e-41b8-b39a-1cb7bad0d496.png)

**Reply by 1715173329** [reply]
Please clean up caches. Try open it in "incognito mode".
